system/Commands/Utilities/Routes/AutoRouterImproved/ControllerMethodReader.php

```diff
@@ -26,6 +26,10 @@ final class ControllerMethodReader
      */
     private string $namespace;
 
+    /**
+     * @var arrray<int, string>
```

Comment:
arrray should be array, I will update when back to PC

Reply:
Oh, sorry!